src/app/shared/components/filter/filter.component.ts

```diff
@@ -374,6 +375,12 @@ export class FilterComponent {
     this._logger.trace('Filters is opened by user, update selected values');
     this._currentFilters = [...this._appliedFilters];
     this._updateSelectedValues(this._currentFilters);
+    if (this._currentFilters.length === 0) {
+      this._initialFilters.forEach(f => {
```

Comment:
@viktor-kyryltsev-kaltura please use (filter: filterItem) or similar instead of "f". We try to keep the code as self explanatory as possible

Reply:
done